src/internal/connector/graph_connector_helper_test.go

```diff
@@ -343,7 +344,22 @@ func checkContact(
 	// Not present in msgraph-beta-sdk/models
 	// assert.Equal(t, expected.GetBusinessPhones(), got.GetBusinessPhones())
 
-	assert.Equal(t, expected.GetCategories(), got.GetCategories(), "Categories")
+	// TODO(ashmrtn): Remove this when we properly set and handle categories in
```

Comment:
@ashmrtn - can you file a ticket for this please?

Reply:
To handle category correctly

Reply:
see:
* #2785